app/Services/Email.php

```diff
@@ -133,8 +134,6 @@ protected function parseLinks($string)
      */
     protected function setupEmail()
     {
-        $leaderboardVars = $this->getLeaderboardVars($this->users);
```

Comment:
also i fixed the issue with the `$topThree` var in PR #245. I had to define the variable outside of the forloop that was seeing values inside for it as @weerd suggested [here](https://github.com/DoSomething/gladiator/pull/244#discussion_r60791479) So it shouldn't be a problem anymore once you rebase with master.